sdk/core/azure_core/perf.yml

```diff
@@ -0,0 +1,47 @@
+trigger: none
+
+pr: none
+
+schedules:
+- cron: "0 0 * * *"
```

Comment:
Not sure it matters, but this probably should be `"0 7 * * *`, unless you want it running at 5PM PST/4PM PDT.